services/src/main/java/org/keycloak/WebAuthnConstants.java

```diff
@@ -65,5 +56,22 @@
     String OPTION_DISCOURAGED = "discouraged";
     String OPTION_NOT_SPECIFIED = "";
 
+    // WebAuthn events
+    interface Events {
```

Comment:
I think I'll remove this interface from here in order to avoid some inconsistencies in external projects. I mean, some of the custom applications might rely on those events, right? Maybe it's only a corner case, but I'd like to not risk it. Or do you think it's worth keeping it there in order to achieve better readability and isolation from those "more important" constants? @mposolda WDYT?